### ISSUE TITLE
Rate boost enabled when switching to Windows OneCore voices for the first time

### ISSUE THREAD

**Issue by tmthywynn8** [nvaccess/nvda#17990]
It appears that if you use the Windows OneCore voices with NVDA for the first time, the speech rate is not set to the default of 50% and rate boost off. Changing the Narrator speed does not have any appreciable differences.

### Steps to reproduce:

1. If nvda.ini has as oneCore subsection, delete it.
2. Start a portable copy of NVDA or run the NVDA installer.

### Actual behavior:

NVDA speaks at a faster rate of speech than the default. It appears that it's not at 50% rate with rate boost enabled as that sounds slightly faster from unmeasured impressions, but it rate boost is definitely enabled as 100 % without rate boost enabled is significantly slower. However, if you open NVDA's speech settings, change the rate with the synth setting ring, toggle rate boost on and off, or simply reselect the Windows OneCore voices, the rate goes back to 50% and without rate boost.

### Expected behavior:

NVDA should be speaking at 50% rate with rate boost unchecked.

### NVDA logs, crash dumps and other attachments:

```log
DEBUG - autoSettingsUtils.autoSettings.AutoSettings._registerConfigSaveAction (02:59:58.084) - MainThread (18424):
registering pre_configSave action: <class 'synthDrivers.oneCore.OneCoreSynthDriver'>
INFO - NVDAHelperLocal (02:59:58.085) - MainThread (18424):
Thread 18424, build\x86\localWin10\oneCoreSpeech.cpp, ocSpeech_initialize, 215:
ocSpeech_initialize

INFO - NVDAHelperLocal (02:59:58.085) - MainThread (18424):
Thread 18424, build\x86\localWin10\oneCoreSpeech.cpp, OcSpeechState::activate, 89:
Activating

INFO - NVDAHelperLocal (02:59:58.089) - MainThread (18424):
Thread 18424, build\x86\localWin10\oneCoreSpeech.cpp, preventEndUtteranceSilence_, 443:
AppendedSilence supported

DEBUGWARNING - synthDrivers.oneCore.OneCoreSynthDriver._isVoiceValid (02:59:58.091) - MainThread (18424):
Missing language data file: %windir%\Speech_OneCore\Engines\TTS\en-CA\MSTTSLocenCA.dat
DEBUGWARNING - synthDrivers.oneCore.OneCoreSynthDriver._isVoiceValid (02:59:58.091) - MainThread (18424):
Could not open registry key HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Speech_OneCore\Voices\Tokens\MSTTS_V110_hiIN_KalpanaM, FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None)
DEBUGWARNING - synthDrivers.oneCore.OneCoreSynthDriver._isVoiceValid (02:59:58.091) - MainThread (18424):
Missing language data file: %windir%\Speech_OneCore\Engines\TTS\en-CA\MSTTSLocenCA.dat
DEBUGWARNING - synthDrivers.oneCore.OneCoreSynthDriver._isVoiceValid (02:59:58.091) - MainThread (18424):
Could not open registry key HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Speech_OneCore\Voices\Tokens\MSTTS_V110_hiIN_HemantM, FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None)
DEBUGWARNING - synthDrivers.oneCore.OneCoreSynthDriver._isVoiceValid (02:59:58.091) - MainThread (18424):
Could not open registry key HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Speech_OneCore\Voices\Tokens\MSTTS_V110_viVN_An, FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None)
DEBUG - speechDictHandler.SpeechDict.load (02:59:58.098) - MainThread (18424):
Loading speech dictionary 'C:\Users\twynn\AppData\Roaming\nvda\speechDicts\voiceDicts.v1\oneCore\oneCore-Microsoft David.dic'...
DEBUG - speechDictHandler.SpeechDict.load (02:59:58.098) - MainThread (18424):
file '...\speechDicts\voiceDicts.v1\oneCore\oneCore-Microsoft David.dic' not found.
DEBUG - autoSettingsUtils.autoSettings.AutoSettings._saveSpecificSettings (02:59:58.099) - MainThread (18424):
Saved settings for OneCoreSynthDriver
INFO - synthDriverHandler.setSynth (02:59:58.099) - MainThread (18424):
Loaded synthDriver oneCore
```

#### NVDA installed/portable/running from source:

Both installed an portable.

#### NVDA version:

2025.1beta1

#### Windows version:

Windows 10 22H2 (OS Build 19045.5608; 64-bit)

#### Name and version of other software in use when reproducing the issue:

Microsoft David (data version 11.0.2016.0129)

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

2019.2.1 appears to have the same behavior.

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes

**Comment by tmthywynn8** [comment]
Ah, I figured out where NVDA is inheriting the default settings when loading the Windows OneCore voices for the first time. It's under [Time & Language➡Speech](ms-settings:speech). If you change the Voice speed slider, NVDA's speed will be immediately effected.
